### PR TITLE
Better testsuites visual hierarchy

### PR DESCRIPTION
**Pull request by jonbell-lot23**
I'm addressing hierarchy issues as described here:

https://linear.app/replay/issue/DES-629/hierarchy-improvements
https://linear.app/replay/issue/DES-628/line-height-is-too-tight-things-are-getting-cropped

Old:
image

New:
image

### CODE REVIEW

**Review by markerikson — APPROVED** [review]

File: src/ui/components/TestSuite/views/TestMetadata/TestItemTree.module.css [hunk 1]
```diff
@@ -1,18 +1,35 @@
...
+  font-size: 13px;
+}
+
+.Tree > .Tree > .Tree > DIV {
+  font-size: var(--font-size-small);
+  color: var(--theme-base-60);
+  text-transform: uppercase;
+  font-weight: bold;
+  line-height: 2rem;
+  margin-top: 0.75rem;
```
Comment (bvaughn) [inline src/ui/components/TestSuite/views/TestMetadata/TestItemTree.module.css:34]: This is pretty fragile.

Comment (bvaughn) [reply src/ui/components/TestSuite/views/TestMetadata/TestItemTree.module.css:34]: @jonbell-lot23  Can you file a linear issue (FE) to revisit this

Comment (bvaughn) [reply src/ui/components/TestSuite/views/TestMetadata/TestItemTree.module.css:34]: btw please tag me on Test Suites CSS PRs :)

Comment (jonbell-lot23) [reply src/ui/components/TestSuite/views/TestMetadata/TestItemTree.module.css:34]: Here's a new ticket: https://linear.app/replay/issue/FE-1475/testsuites-hierarchy-needs-better-css

**Comment by bvaughn** [comment]
Looks like this PR caused a regression: FE-1480

**Comment by bvaughn** [comment]
Reverted in [6710927](https://github.com/replayio/devtools/commit/67109276d27b998d81d5854bf46e350021bd2035)

**Pull request merged**